src/smpdflib/plots.py

```diff
@@ -132,6 +134,51 @@ def plot_pdfs(pdfsets, Q, base_pdf = None, flavors=None, photon=False):
 
         yield (fl,),fig
 
+def plot_pdfcorr(pdfsets, Q, base_pdf=None, flavors=None, photon=False):
+
+    if flavors is None:
+        flavors = PDF.make_flavors(photon=photon)
+
+    nflavors = len(flavors)
+    xgrid = PDF.make_xgrid()
+
+    resultlists = []
+    for pdf in pdfsets:
+        mean, replicas = pdf.grid_values(Q, xgrid, flavors)
+        replicas = replicas.reshape(replicas.shape[0],
+                                    replicas.shape[1]*replicas.shape[2])
+        if pdf == base_pdf:
+            base_results = replicas
+        resultlists.append(replicas)
+
+    for pdf, replicas in zip(pdfsets, resultlists):
+        fig = plt.figure()
+
+        corrmat = np.corrcoef(replicas, rowvar=False)
+        if base_pdf:
+            corrmat -= np.corrcoef(base_results, rowvar=False)
+
+        plt.imshow(corrmat, cmap=plotutils.spectral_cm, vmin=-1, vmax=1)
+        plt.grid(False)
+        if base_pdf:
+            plt.title("Correlations at $Q=%.2f$ GeV for\n%s-%s" %
+                      (Q, pdf.label, base_pdf.label), fontsize=15)
```

Comment:
Is fontsize=15 a good default? Could it go in the style or is just good for this plot?